src/core/console.c

```diff
@@ -68,10 +68,10 @@ void Con_SetExecFunction(int(*exec_cmd)(char *ch))
 
 void Con_InitGlobals()
 {
-    con_base.background_color[0] = 1.0f;
-    con_base.background_color[1] = 0.9f;
-    con_base.background_color[2] = 0.7f;
-    con_base.background_color[3] = 0.4f;
+    con_base.background_color[0] = 0;
```

Comment:
Shouldn't this be 255?

Reply:
I set the values that are currently in use in the repo `config.lua` file (`background_color = {r = 0, g = 0, b = 0, a = 200};`), but you can use the old color values if you prefer.